client/app/templates/json-api-errors.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if errors}}
```

Comment:
I know you can pass in properties to components, maybe that's a better usage:
https://guides.emberjs.com/v2.2.0/components/passing-properties-to-a-component/